website/docs/r/group_profile.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "turbot"
+title: turbot
+template: Documentation
+page_title: "Turbot: turbot__group_profile"
+nav:
+  title: turbot__group_profile
+---
+
+# turbot_profile
+
+The `Turbot group profile` resource adds support for creating group profile under a directory type. It is used to create, manage and delete group profiles.
+
+## Example Usage
+
+**Creating Your First group profile**
+
+```hcl
+resource "turbot_group_profile" "test" {
+  title               = "terraform test group profile"
+  directory           = "112233445566"
```

Comment:
AKAs are better than IDs in most documentation, please use a sample AKA here instead.